plugins/inventory/proxmox.py

```diff
@@ -360,7 +362,17 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
                         agent_iface_key = self.to_safe('%s%s' % (key, "_interfaces"))
                         properties[agent_iface_key] = agent_iface_value
 
-                if config not in plaintext_configs and not isinstance(value, int) and all("=" in v for v in value.split(",")):
+                if config == "lxc":
+                    out_val = {}
+                    for key_list in value:
+                        k, v = key_list
```

Comment:
Why not simply write:
```suggestion
                    for k, v in value:
```
I assume that the config is a list of lists, where every inner list has exactly two elements?

Reply:
Exact.

Reply:
Integrated.